model/src/main/java/org/projectnessie/api/TreeApi.java

```diff
@@ -233,7 +233,19 @@ LogResponse getCommitLog(
           Integer maxRecords,
       @Parameter(description = "pagination continuation token, as returned in the previous LogResponse.token")
       @QueryParam("pageToken")
-          String pageToken)
+          String pageToken,
+      @Parameter(description = "The author of a commit. This is the original committer")
+      @QueryParam("author")
+      String author,
+      @Parameter(description = "The entity that performed the commit")
```

Comment:
I think the parameter descriptions should be a bit more descriptive. 

eg Filter the log on the author that specified the commit. Can take multiple authors and a regular expression

Similarily there should be a few words in the `@Operation` above to discuss the filters

Reply:
fixed